tensorflow/java/src/gen/cc/ops/op_generator.cc

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
@kbsriram  is about to send another PR where an annotation processor written in Java will generate the Ops API file (from the generated operations and others found in the classpath). We planned to put this generator under src/gen/java, the reason why having a src/gen/cc here. Does it make more sense to you?

For the /ops subfolder, I was planning to split the source code of the generator in a few files so it would make sense to remove it only if you are confident that we won't have other generators written in C++ under the Java package.